Formula/j/jpeg-xl.rb

```diff
@@ -54,12 +55,17 @@ class JpegXl < Formula
   def install
     ENV.append_path "XML_CATALOG_FILES", HOMEBREW_PREFIX/"etc/xml/catalog"
     resources.each { |r| r.stage buildpath/"third_party"/r.name }
+    # jpegli.cmake looks for these `jpeg-turbo` headers at hardcoded path
+    cp Formula["jpeg-turbo"].include/"jconfig.h",
+       buildpath/"third_party/libjpeg-turbo/jconfig.h.in"
+    cp [Formula["jpeg-turbo"].include/"jpeglib.h", Formula["jpeg-turbo"].include/"jmorecfg.h"],
```

Comment:
I was tempted to write `Formula["jpeg-turbo"].include/["jpeglib.h", "jmorecfg.h"]` but that doesn't work (No implicit conversion from Array to String). Ruby actually handled `cp Array[src] dest` the intuitive way, nice.